DirectEmployers/MyJobs
# [PD-3663] ie fix multifilters

## What's this PR do?
Fixes the multifilters in IE

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of site traffic for analytics

## How should this be manually tested?
Pull down the PR. Go into site traffic and check the searchbox to make sure that it looks normal

## Any background context you want to provide?
